src/test/java/com/lebediev/movieland/dao/jdbc/integrationtests/JdbcMovieDaoITest.java

```diff
@@ -80,10 +80,47 @@ public void testGetMoviesByGenreIdOrdered() {
     public void testGetMovieById() {
         List <Movie> movieList = jdbcMovieDao.getAll(new SortParams());
         assertNotEquals(movieList.size(), 0);
-        int movieId = movieList.get(0).getMovieId();
-        Movie actual = jdbcMovieDao.getMovieById(movieId);
+        int id = movieList.get(0).getId();
+        Movie actual = jdbcMovieDao.getMovieById(id);
         assertNotEquals(actual, 0);
     }
 
+    @Test
+    public void testAdd(){
+        Movie movie = new Movie();
+        movie.setNameNative("ITest movie");
+        movie.setYearOfRelease(1900);
+        jdbcMovieDao.add(movie);
+        int result = jdbcTemplate.queryForObject("SELECT COUNT(*) FROM movie WHERE nameNative = 'ITest movie' AND yearOfRelease = 1900", Integer.class);
+        assertEquals(result, 1);
+        jdbcTemplate.update("DELETE FROM movie WHERE nameNative = 'ITest movie' AND yearOfRelease = 1900");
+    }
+
+    @Test
+    public void testUpdate(){
+        Movie actual = new Movie();
+        actual.setNameNative("before update");
+        actual.setYearOfRelease(2000);
+        jdbcMovieDao.add(actual);
+        int id = jdbcTemplate.queryForObject("SELECT id FROM movie WHERE nameNative = 'before update' AND yearOfRelease = 2000", Integer.class);
+        assertNotEquals(id, null);
```

Comment:
when id will be null?

Reply:
good point - if no rows selected then exception will be thrown, fixed, thanks )